### DISCUSSION TITLE
Native MKB Support Removed?

### DISCUSSION THREAD

**Discussion by Sus002** [redphx/better-xcloud#416] [General]
"Hey, I've been using Better xCloud for more than 6 months now, and for some time after that, I have been using it with Native MKB Support. Every time there was an update, I would just go and change the code where it said Enable_NATIVE_MKB_Support=false, and then choose the game. It was working just fine. However, when I recently updated to the latest version, it vanished. Was it removed? If not, can you show me how to activate it again? By the way, thanks for the great work."

**Comment by KingNothing81** [comment]
Since version 4.4.0 there is an option for this in the script settings…
https://github.com/redphx/better-xcloud/releases/tag/v4.4.0

<img width="459" alt="PNG-Bild" src="https://github.com/redphx/better-xcloud/assets/146887651/7d8d793a-0e85-47a8-ae89-e8c6df4295d9">

So I guess as long as you don’t disable it, native MKB should work for those games that have support for it. There is a list here:

https://news.xbox.com/en-us/2024/05/15/xbox-may-update-2024/

If that’s not the case for you, maybe you should create an issue about this.

https://github.com/redphx/better-xcloud/issues

**Reply by Sus002** [reply]
Oh, I see. Thanks for answering, but it was actually removed. The game I used to play with native mouse and keyboard was Flight Simulator, and just by activating it in the script, it would work without having the new support from Microsoft. So, all I guess I gotta do is wait for Flight Simulator to be added to the official mouse and keyboard support. Even though I just downgraded to the last version it was available, I just changed the option from false to true, and the native mouse and keyboard support works on version 3.5.2, even on games that are not on the list. Thanks, though!

**Comment by redphx** [comment]
Test this:  
1. Uninstall Better xCloud script
2. Install this, then check its source for instruction: https://github.com/redphx/better-xcloud/raw/typescript/scripts/custom-flags.user.js
3. Reinstall Bx

**Reply by Sus002** [reply]
Wow! you the goat, W's for you fr, it worked, keep up the awesome work with better xcloud, you're helping more then you know.

**Comment by redphx** [comment]
<img width="442" alt="image" src="https://github.com/user-attachments/assets/fdcb4161-45ea-4329-9a3b-3686a431f263">

Added setting

**Comment by AshXD007** [comment]
does not work, no response no matter what I try
